src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -1060,11 +1063,19 @@ def test_edit_xss(self):
                                   "<script>alert('awesome')</script>")
         self.addon.save()
         response = self.client.get(self.url)
+        assert response.status_code == 200
         doc = pq(response.content)
 
         assert doc('#edit-addon-details span[lang]').html() == (
             "This<br/><b>IS</b>&lt;script&gt;alert('awesome')&lt;/script&gt;")
 
+        response = self.client.get(self.details_edit_url)
+        assert response.status_code == 200
+
+        assert '<script>' not in response.content
+        assert ('This\n&lt;b&gt;IS&lt;/b&gt;&lt;script&gt;alert(&#39;awesome'
+                '&#39;)&lt;/script&gt;</textarea>') in response.content
```

Comment:
Just extra paranoia on my part since we weren't testing `GET` for `self.details_edit_url` before.